.github/ISSUE_TEMPLATE/issue.yaml

```
@@ -0,0 +1,76 @@
+name: Request support
+description: I need to request support from the Software Engineering Team
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This is a form to request support from the University of Colorado Department of Biomedical Informatics **Software Engineering Team** (SET). [⭐ List of our services and current team members ⭐](https://cu-dbmi.github.io/set-website/about)
+
+        **This form is not meant to be a barrier to support**, it is only meant to save time and allow us to better help you by asking a standard set of questions and tracking requests in an organized way. If you already have an established contact and agreement with a specific SET developer, you can continue to communicate directly with them.
+
+        If you don't have a GitHub account, you can instead use [this Google Form](https://docs.google.com/forms), which will create an request here automatically. You can look at what support others have requested [here](https://github.com/CU-DBMI/set-intake/issues). Note that these requests are publicly visible. If you need to make a private request, please contact [Audrey Wen](mailto:audrey.wen@cuanschutz.edu) with the same info as below.
+
```

Wondering if it would make sense to encourage folks to have or use a GitHub account to receive notifications. Unsure if there's a way to do the reverse (GitHub issue update --to--> Google form email field / validated intake requirement).

```suggestion

Please note: using a GitHub account in association with these requests enables you to use [GitHub notifications](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/about-notifications) to receive updates as they occur here.
```

---

I think this is a good idea. Can we change "please note" to "tip" or "we recommend".